### PR TITLE
40-m1-implement-a-folder-structure: moved functions to modules

### PR DESCRIPTION
**Pull request by sebastianpiq**
Created a folder structure for the library using inputValidator, parser, and CMCDvalidator as modules

### Folder structure:

  - src
    - inputValidator
      - index.js
    - parser
      - index.js
    - CMCDValidator
      - index.js
    - utils
      - constants.js
      - helpers.js
    - index.js

Each function was moved to its corresponding module

### CODE REVIEW

File: src/parser/index.js [hunk 1]
```diff
@@ -1,14 +1,13 @@
-/* eslint-disable import/prefer-default-export */
```
[the remark this answers is not in the document]
Comment (arrivillagaTomas) [reply src/parser/index.js:1]: Scratch that -` index.js` I believe should hold the functionality code. The test functions and the test index.js should go in their own folder inside the tests subdirectory. 
@pollingerMaxi @CamiloBorba09  @JoaquinBCh @sebastianpiq Thoughts?

[the remark this answers is not in the document]
Comment (JoaquinBCh) [reply src/parser/index.js:1]: I think we shouldn't have logic inside index.js we should only use it's to export.
For example inside parser we should have this structure:

- parse
 - queryToJson
   - queryToJson.js (query to json parser logic)
   - index.js (export { default } from './queryToJson.js')
 - headerToJson
   - headerToJson.js (header to json parser logic)
   - index.js (`export { default } from './headerToJson.js'`)
  - index.js 
    `import parseQueryToJson from './queryToJson';
     import parseHeaderToJson from './headerToJson';
     export { parseQueryToJson, parseHeaderToJson };`

[the remark this answers is not in the document]
Comment (CamiloBorba09) [reply src/parser/index.js:1]: I agree with you @JoaquinBCh . The only question that I have is, where are we going to put the class or the fuctions of the library?

[the remark this answers is not in the document]
Comment (arrivillagaTomas) [reply src/parser/index.js:1]: @CamiloBorba09 those can be on a top level index.js file.
@JoaquinBCh That's works for me 👍 . My main point was the location of the tests.

**Review by arrivillagaTomas — APPROVED** [review]

**Review by rodrigo-mira — APPROVED** [review]

**Review by JoaquinBCh — APPROVED** [review]

**Review by Rspillere — APPROVED** [review]

**Review by mcamardon — APPROVED** [review]

**Pull request merged**